tests/suites/test_suite_psa_crypto.data

```diff
@@ -6320,6 +6320,10 @@ PSA key derivation: HKDF-Expand SHA-1, request too much capacity
 depends_on:PSA_WANT_ALG_HKDF_EXPAND:PSA_WANT_ALG_SHA_1
 derive_set_capacity:PSA_ALG_HKDF_EXPAND(PSA_ALG_SHA_1):255 * PSA_HASH_LENGTH(PSA_ALG_SHA_1) + 1:PSA_ERROR_INVALID_ARGUMENT
 
+PSA key derivation: TLS 1.2 PSK-to-MS, request too much capacity
+depends_on:PSA_WANT_ALG_SHA_256:PSA_WANT_ALG_TLS12_PSK_TO_MS
+derive_set_capacity:PSA_ALG_TLS12_PSK_TO_MS(PSA_ALG_SHA_256):48U + 1U:PSA_ERROR_INVALID_ARGUMENT
+
```

Comment:
I have added the testcases but they are failing in the 32-bit compiler tests. Should I comment out the test cases or is there an alternative solution?

Reply:
The tests need to run only on platforms where `SIZE_MAX >= 2^32-1*hLen`, and in practice we can use `SIZE_MAX >= 0xffffffffffffffff` because we don't care about hypothetical platforms where `SIZE_MAX` is between 2^33 and 2^63. You can use `SIZE_MAX>=0xffffffffffffffff` in a `depends_on` statement.